lib/utils.pm

```diff
@@ -101,6 +102,10 @@ sub unlock_if_encrypted {
     }
 }
 
+sub turn_off_packagekit() {
```

Comment:
what can I do is remove turn_off_pacakgekit() and move the code to live_installation, fine with that?